po/it_IT.po

```diff
@@ -0,0 +1,1554 @@
+#, perl-brace-format
+msgid "Cannot find {script} template"
+msgstr "Non trovo il template {script}"
+
+#: lib/App/Sqitch/Command/add.pm:218
+#, perl-brace-format
+msgid "Option --{opt} has been deprecated; use \"--{with} {val}\" instead"
+msgstr "L'opzione --{opt} è deprecata, usa \"--{with} {val}\" al suo posto"
+
+#: lib/App/Sqitch/Command/add.pm:236
+#, perl-brace-format
+msgid "Option --{opt} has been deprecated; use \"--use {key}={val}\" instead"
+msgstr "L'opzione --{opt} è deprecata, usa \"--use {key}={val}\" al suo posto"
+
+#: lib/App/Sqitch/Command/add.pm:265
+#, perl-brace-format
+msgid "Directory \"{dir}\" does not exist"
+msgstr "La directory {dir} non esiste"
+
+#: lib/App/Sqitch/Command/add.pm:270
+#, perl-brace-format
+msgid "\"{dir}\" is not a directory"
+msgstr "\"{dir}\" non è una directory"
+
+#: lib/App/Sqitch/Command/add.pm:322
+#, perl-brace-format
+msgid ""
+"Name \"{name}\" identifies a target; use \"--change {name}\" to use it for "
+"the change name"
+msgstr ""
+"Il nome \"{name}\" identifica un target, usa \"--change {name}\" per "
+"effettuare il cambio di nome"
+
+#: lib/App/Sqitch/Command/add.pm:370
+msgid "add"
+msgstr "aggiungi"
+
+#: lib/App/Sqitch/Command/add.pm:388
+#, perl-brace-format
+msgid "Added \"{change}\" to {file}"
+msgstr "\"{change}\" aggiunta al file {file}"
+
+#: lib/App/Sqitch/Command/add.pm:407
+#, perl-brace-format
+msgid "Skipped {file}: already exists"
+msgstr "Salto {file}: esistente"
+
+#: lib/App/Sqitch/Command/add.pm:418 lib/App/Sqitch/Command/bundle.pm:141
+#: lib/App/Sqitch/Role/TargetConfigCommand.pm:261
+#, perl-brace-format
+msgid "Error creating {path}: {error}"
+msgstr "Errore durante la creazione di {path}: {error}"
+
+#: lib/App/Sqitch/Command/add.pm:435 lib/App/Sqitch/Command/add.pm:465
+#: lib/App/Sqitch/Plan.pm:135 lib/App/Sqitch/Plan.pm:588
+#: lib/App/Sqitch/Plan.pm:969 lib/App/Sqitch/Plan/Line.pm:107
+#: lib/App/Sqitch/Role/TargetConfigCommand.pm:325
+#, perl-brace-format
+msgid "Cannot open {file}: {error}"
+msgstr "Non posso aprire {file}: {error}"
+
+#: lib/App/Sqitch/Command/add.pm:443
+#, perl-brace-format
+msgid "Error executing {template}: {error}"
+msgstr "Errore nell'esecuzione di {template}: {error}"
+
+#: lib/App/Sqitch/Command/add.pm:455
+#: lib/App/Sqitch/Role/TargetConfigCommand.pm:336
+#, perl-brace-format
+msgid "Error closing {file}: {error}"
+msgstr "Errore nella chiusura di {file}: {error}"
+
+#: lib/App/Sqitch/Command/add.pm:459 lib/App/Sqitch/Command/bundle.pm:134
+#: lib/App/Sqitch/Command/init.pm:195
+#: lib/App/Sqitch/Role/TargetConfigCommand.pm:255
+#: lib/App/Sqitch/Role/TargetConfigCommand.pm:341
+#, perl-brace-format
+msgid "Created {file}"
+msgstr "{file} creato"
+
+#: lib/App/Sqitch/Command/bundle.pm:99
+msgid ""
+"Use of --to or --from to bundle multiple targets is not recommended.\n"
+"Pass them as arguments after each target argument, instead."
+msgstr ""
+"L'uso di --to o --from per creare un bundle con piu' target è sconsigliato.\n"
```

Comment:
"L'uso di --to o --from per creare un bundle con **più** target è sconsigliato.\n"